### PR TITLE
Rill CLI changes 

### PR DESCRIPTION
**Pull request by rakeshsharma14317**

### CODE REVIEW

File: cli/cmd/source/add.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+		Use:   "add",
+		Short: "Add source to the project",
+		Long:  `Add source to the project`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			fmt.Println("add called")
+			return nil
+		},
+	}
+
+	addCmd.Flags().StringVarP(&sourcePath, "source-path", "p", ".", "Source path for Rill")
```
Comment (begelundmuller) [inline cli/cmd/source/add.go:22]: This should be a required arg, probably a positional arg and not a flag, e.g. `rill source add /path/to/file.csv`

File: cli/cmd/source/drop.go [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+		Use:   "drop",
+		Short: "Drop source to the project",
+		Long:  `Drop source to the project.`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			fmt.Println("drop called")
+			return nil
+		},
+	}
+
+	dropCmd.Flags().StringVarP(&sourcePath, "source-path", "p", ".", "Source path for Rill")
```
Comment (begelundmuller) [inline cli/cmd/source/drop.go:22]: Should be a positional arg and a name, e.g. `rill source drop ad_bids`

**Review by begelundmuller — APPROVED** [review]
Go for the `rill source ...` approach, it looks better! Made a few comments, but approving so you can merge when you've addressed them

File: cli/cmd/source/add.go (hunk 1 above)
Comment (rakeshsharma14317) [reply cli/cmd/source/add.go]: Make sense, keep it as args with check of MinimumNArgs(1) so that it can support multiple if required in future

**Pull request merged**